cmd/thanos/bucket.go

```diff
@@ -582,10 +582,21 @@ func (t Table) Less(i, j int) bool {
 }
 
 func compare(s1, s2 string) bool {
+	// Values can be either Time, Duration or other.
```

Comment:
comma-delimited integer also needs to be handled, for `#SAMPLES`, etc.

see https://github.com/thanos-io/thanos/issues/2414#issuecomment-612554861

Reply:
This was my initial comment where I thought there were only 3 different possible  string 'types', it was only a reminder. I can update it or remove it, what do you think?

Reply:
I mean, they are being handled despite of not being specified in the comment

Reply:
We could update this comment to fit the reality.